ide-setup.md

```diff
@@ -51,7 +51,19 @@ To install the plugins:
    directory, without the `bin` subdirectory. For example, `c:\Users\obiwan\flutter` / `/Users/obiwan/flutter`.
 1. Click **OK**
 
-## Using the IDEs for Flutter
+### Using the IDEs for Flutter
```

Comment:
"Using IDEs and editors for Flutter"

Reply:
The page linked to here is currently IDEs only